ceresdb-protocol/src/main/java/io/ceresdb/models/Value.java

```diff
@@ -72,7 +72,7 @@ public DataType getDataType() {
         return type;
     }
 
-    public Object getValue() {
+    public Object getAnyValue() {
```

Comment:
I guess `getValue` or `getRawValue` is better?

Reply:
This is to be consistent with golang and rust